From 315adfe1cd76fd95645c99a985c1eeaacb919524 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:32:12 -0400
Subject: [alsa] hda: if statement not indented

Message-id: <140972823573-294-patchset-jkysela@redhat.com>
Patchwork-id: 90990
O-Subject: [RHEL 7.1 PATCH 293/442] ALSA: hda - if statement not indented
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

The "break;" should be indented.

Signed-off-by: Dan Carpenter <dan.carpenter@oracle.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>
Signed-off-by: Jaroslav Kysela <perex@perex.cz>

(picked from kernel: d576422eda7fc999bf7c9862fa0a75bf0323417f)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/hda/hda_generic.c b/sound/pci/hda/hda_generic.c
index 1613388..589e47c 100644
--- a/sound/pci/hda/hda_generic.c
+++ b/sound/pci/hda/hda_generic.c
@@ -3722,7 +3722,7 @@ static void parse_digital(struct hda_codec *codec)
   } else {
    spec->multiout.slave_dig_outs = spec->slave_dig_outs;
    if (nums >= ARRAY_SIZE(spec->slave_dig_outs) - 1)
-   break;
+    break;
    spec->slave_dig_outs[nums - 1] = dig_nid;
   }
   nums++;
-- 
1.7.1